packages/@ember/object/tests/reopen_test.js

```diff
@@ -1,12 +1,12 @@
-import { get } from '@ember/-internals/metal';
-import EmberObject from '../../../lib/system/object';
+import { get } from '@ember/object';
+import CoreObject from '@ember/object/core';
```

Comment:
Hmm, are you *sure* about this change? Seems like we should probably prefer to keep it the same for *this* change, and then if we think this is correct, update in a follow-on PR?

Reply:
A few lines down, the tests claim they're for CoreObject, which I think is why I changed it.